apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -436,6 +436,26 @@ static Extension createExtensionDate(CcwCodebookInterface ccwVariable, BigDecima
     return extension;
   }
 
+  /**
```

Comment:
missing a javadoc description here but also wondering (see below) whether we can reduce some duplication here.

Reply:
Updated javadoc. Since we don't have to map to V1 anymore the duplication is gone. The parameter we are trying to map is a localDate and the existing createExtensionDate method takes BigInt, so I felt creating a separate method made sense.